spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,43 @@
+RSpec.describe 'POST api/v1/targets', type: :request do
```

Comment:
Maybe we can add more tests, ie:
- Target has the correct user
- Target was saved
- With some missing/incorrect value